CondFormats/PPSObjects/src/classes_def.xml

```diff
@@ -68,4 +68,17 @@
     <class name="LHCInterpolatedOpticalFunctionsSetCollection"/>
 
     <class name="PPSDirectSimulationData"  class_version="0"/>
-</lcgdict>
\ No newline at end of file
+
+    <class name="PointErrors"/>
+    <class name="SelectionRange"/>
+    <class name="RPConfig"/>
+    <class name="SectorConfig"/>
+    <class name="Binning"/>
+    <class name="std::vector<PointErrors>"/>
+    <class name="std::map<unsigned int, std::vector<PointErrors>>"/>
+    <class name="std::pair<unsigned int, std::vector<PointErrors>>"/>
+    <class name="std::map<unsigned int, SelectionRange>"/>
+    <class name="std::pair<unsigned int, SelectionRange>"/>
+    <class name="PPSAlignmentConfig" class_version="0"/>
+
+  </lcgdict> 
```

Comment:
There is a newline missing here, since the PR is not merged yet, would you @MatiXOfficial like to fix that?

Reply:
Sure, I can fix it, but is it a good idea? The main PR (#34844) has already been merged into master.

Reply:
@perrotta @qliphy what do you think?

Reply:
> @perrotta @qliphy what do you think?

While not being pleasant to see while reading the code on github, I don't see any issue because of this missing newline at the end of the file: it was as such even before this fix, and it was already merged as such in the master. Also with the intention of having as similar as possible code in the master and in the 12_0 backport, I would avoid implementing this possible aesthetical fix here.